libs/judicial-system/formatters/src/lib/formatters.ts

```diff
@@ -160,12 +162,29 @@ export function formatCustodyRestrictions(
     return 'Sækjandi tekur fram að gæsluvarðhaldið sé án takmarkana.'
   }
 
+  const formattedValidToDateAndTime = `${formatDate(
+    validToDate,
+    'PPPp',
+  )?.replace(' kl.', ', kl.')}`
+
+  const formattedIsolationToDateAndTime = `${formatDate(
```

Comment:
Why is there a mismatch in the wording and date format here and here: https://github.com/island-is/island.is/pull/4263/files#diff-b2788662cb5317437f7ba480f353c0d321e3181497eb8d1accfed00b066ca25eR111

Day and time formatting
```skal sæta einangrun ekki lengur en til {date.replace('dagur,', 'dagsins')?.replace(' kl.', ', kl.')}```
vs
Only time formatting
```skuli sæta einangrun til {date.replace(' kl.', ', kl.')}```

Reply:
No particular reason, but there is no reason they should be the same either. These are two different sentences.

The general pattern has been to use a short version in SMS and long version else where. I will modify this text to follow that convention.